src/dct/templates/Template.lua

```diff
@@ -145,8 +146,7 @@ local function checkbldgdata(keydata, tpl)
 
 	for _, bldg in ipairs(tpl[keydata.name]) do
 		local bldgdata = {}
-		bldgdata.countryid = 0
-		bldgdata.category  = enum.UNIT_CAT_SCENERY
+		bldgdata.category = enum.UNIT_CAT_SCENERY
```

Comment:
Scenery doesn't have a country associated anymore, so coalitions cannot be detected from it anymore. This is intentional, as countryid 0 would auto-assign it to Russia.